core/types/gen_log_json.go

```diff
@@ -10,16 +10,18 @@ import (
 	"github.com/ethereum/go-ethereum/common/hexutil"
 )
 
+var _ = (*logMarshaling)(nil)
```

Comment:
? where is it used?

Reply:
don't know, it's auto generated file
I needed to autogenerate because of that with pending logs, there are some values that can be null and was required before the fix (tx index and log index)